zarr/core.py

```diff
@@ -2518,7 +2518,7 @@ def bytestr(n):
         # filters
         if self.filters:
             for i, f in enumerate(self.filters):
-                items += [("Filter [%s]" % i, repr(f))]
```

Comment:
Only the first argument `i` was printed, but I think the main intent mostly was to print `repr(f)`. Please advise.